ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/server/computation/RecoveryTest.java

```diff
@@ -767,7 +859,8 @@ public void shouldRecoverDropWithSourceConstraintsFromOldMetastore() {
     assertThat(server1.ksqlEngine.getMetaStore().getAllDataSources(), hasKey(SourceName.of("B")));
 
     // Recovered server has only stream 'B'
```

Comment:
Update comment?

Reply:
Added better comments

Reply:
Sorry, to be more precise, I had just noticed that line 862 in RecoveryTest.java says the recovered server only has stream 'B'.